Adyen.docc/DropIn.md

```diff
@@ -31,63 +56,51 @@ let applePayment = try ApplePayPayment(countryCode: "US",
                                        currencyCode: "USD",
                                        summaryItems: summaryItems)
 
-let configuration = DropInComponent.Configuration(apiContext: apiContext)
+let dropInConfiguration = DropInComponent.Configuration(context: context)
 configuration.applePay = .init(payment: applePayment,
```

Comment:
configuration  -> dropInConfiguration

Reply:
We have configuration name for session above, so `dropInConfiguration` makes sense, but I will rename the next line too.